server/graphql/v2/query/ExpensesQuery.ts

```diff
@@ -77,6 +81,25 @@ const ExpensesQuery = {
       where['CollectiveId'] = collective.id;
     }
 
+    // Add search filter
+    if (args.searchTerm) {
+      where[Op.or] = [
+        { description: { [Op.substring]: args.searchTerm }},
+        { tags: { [Op.overlap]: [args.searchTerm] }},
+        { '$fromCollective.name$': { [Op.substring]: args.searchTerm }},
+        { '$items.description$': { [Op.substring]: args.searchTerm }}
+      ];
+
+      include = [
+        { model: models.Collective, as: 'fromCollective' },
+        { model: models.ExpenseItem, as: 'items', duplicating: false }
```

Comment:
By adding `attributes: []` we can make sure that the `SELECT` won't include any fields, thus making the query simpler and improving the RAM usage.

```suggestion
        { model: models.Collective, as: 'fromCollective', attributes: [] },
        { model: models.ExpenseItem, as: 'items', duplicating: false, attributes: [] }
```

Reply:
Done. :+1: